homeassistant/components/nina/__init__.py

```diff
@@ -80,7 +82,7 @@ def __init__(
 
         super().__init__(hass, _LOGGER, name=DOMAIN, update_interval=SCAN_INTERVAL)
```

Comment:
Side note: `self.warnings` above doesn't seem used. Maybe we can clean that attribute up?

Reply:
Yes it is no longer needed, I remove it.